pyproject.toml

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[tool.isort]
+profile = "black"
+line_length = 119
+ignore_whitespace = true
+py_version = 38
```

Comment:
Can you inline some docstrings so that we know what to upgrade when you start using py3.9 as default min. Same for other args

Reply:
Added comments